pages/instructorIssues/instructorIssues.ejs

```diff
@@ -181,12 +186,23 @@
               <div class="d-block">
                 <strong><%= row.question_qid %></strong>
                 (<a href="<%= urlPrefix %>/question/<%= row.question_id %>/?variant_id=<%= row.variant_id %>">instructor
-                  view</a><% if (row.instance_question_id) { %>,
+                  view</a><% if (row.instance_question_id && row.show_user) { %>,
```

Comment:
I'm assuming that the instructor view is still accessible, even if we don't have student data access. But I think that maybe it shouldn't be accessible? It will contain the student's answer, including uploaded files that might contain PII (e.g., a PDF with a student's name on it). I think we should change the access control for the instructor question page to check whether we have access to the variant (and also remove the instructor view link here).